src/pysdccc/_cli.py

```diff
@@ -0,0 +1,133 @@
+import contextlib
+import io
+import pathlib
+import subprocess
+import sys
+import tempfile
+import zipfile
+
+import httpx
+
+from pysdccc import _download, _runner
+
+try:
+    import click
+except ImportError as import_error:
+    msg = "Cli not installed. Please install using 'pip install pysdccc[cli]."
```

Comment:
missing word: Please install **it** using ...

Reply:
done